components/chart/index.tsx

```diff
@@ -172,6 +187,9 @@ function Chart({
             changeZoomLevel={changeZoomLevel}
           />
         </div>
+        < DescriptionDrawer title={selectedModule?.title} open={description != ''} onClose={toggleDrawer}>
```

Comment:
```suggestion
        <DescriptionDrawer module={selectedModule} open={!!selectedModule} onClose={() => setSelectedModule(undefined)}>
```
And you can get the title, description from the module itself inside the DescriptionDrawer